crates/core/src/factories/mem_peer_store/test.rs

```diff
@@ -23,68 +22,6 @@ fn unsneak_url(u: &Url) -> String {
     u.peer_id().unwrap().into()
 }
 
-#[derive(Debug, Default)]
```

Comment:
Moved this to `test_utils` under this module and made public for re-use in fetch tests. Is that a good place for it?

Reply:
I'm okay with this for now. I may move this into the test_utils crate later when I've got some more agent building tests to implement : )